environments/icds-new/known_hosts

```diff
@@ -160,3 +160,5 @@
 10.247.164.9, ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCvvWgmO3RUm9L6x3Uc//2ij/2LtccOo3y4JH8JVSxDQCl+t0qjjMLOrEmFEQUCjFdhs3SQqNFlbbPmn7rs2TV4QcQj585GOsWYkn8MOEEoB27bkgIQEbbpGGBHM5TexRqdFLUn6vGaPExk0Esp4uRkXfoLuMDfLpgLTXjt0yllmmAvblNpt6JumKcaY4wL9SeX9vSaop+ziR4RHcbuYOQwHoAR0ZXLvyt55bxiE2bfobq9JEwpHP0toZpR75vVDD6YyD1i4z43nexp+lZyNZUBnEBWdO+8V0TUYBqMS6yBW481pBZvhZNBlGca4W1+jv6phDYAIDCmlYvVFiwRmiEf
 10.247.164.88 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBPJlHghVkxEe3vpJKEx/c8FGJtK03+djz2O7ZGnblNBb+aTd07Rd8z0eZpNWH2scQDQWTqXREjQNjrDyI/cuC8Q=
 10.247.164.89 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBPJlHghVkxEe3vpJKEx/c8FGJtK03+djz2O7ZGnblNBb+aTd07Rd8z0eZpNWH2scQDQWTqXREjQNjrDyI/cuC8Q=
+|1|KHP4oyBpuTDesamvx10KT12/K4U=|mS5tU7+dVbyBWV1lJfeuRjn8Zn0= ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBHw4iz18hVc0aoHht/VtgESpB24w5nCzd+yjfPPy6DsdC+9xWcqLXgzL7mEt9lSV50wVQG1QK23D89HQMo8Pk+I=
+|1|uMAOQX0NfQk871bDk4cNUrgVho4=|g+72r03K55Q42wmGKGbI6bijc/o= ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBPJlHghVkxEe3vpJKEx/c8FGJtK03+djz2O7ZGnblNBb+aTd07Rd8z0eZpNWH2scQDQWTqXREjQNjrDyI/cuC8Q=
```

Comment:
looks like these were added by sshing into the host directly rather than using the `add-ssh-keys` command